tests/store/tracking/test_file_store.py

```diff
@@ -2970,12 +2956,6 @@ def test_search_traces_filter(generate_trace_infos):
         "trace.status = 'ERROR'",
         trace_infos[2:5][::-1],
     )
-    _validate_search_traces(
-        store,
-        [exp_id],
-        "attributes.status IN ('IN_PROGRESS', 'OK')",
```

Comment:
Oh I thought we still support `IN` for `attributes` - we support it for runs - probably a bug?
https://github.com/mlflow/mlflow/blob/master/mlflow/utils/search_utils.py#L141